packages/apm/src/span.ts

```diff
@@ -13,18 +13,24 @@ import {
   uuid4,
 } from '@sentry/utils';
 
+const INITIAL_TIME = Date.now();
+
+const performanceFallback: Pick<Performance, 'now'> = {
+  now(): number {
+    return INITIAL_TIME - Date.now();
```

Comment:
I think storing the initial time is unnecessary.
The point of using performance.now is to compute durations t1-t0; it doesn't matter to subtract yet INITIAL_TIME. And as a fallback it is limited anyway, since it is has no guarantee on being monotonic, unlike the real performance.now.

Reply:
`performance.now` returns a delta from start till now. We need a fallback to do the same.